Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/LoadTestDataDocumentBean.java

```
@@ -291,24 +319,16 @@ public void setPatientId(String patientIdentifierIso) {
 
         if (StringUtils.isNotBlank(patientIdentifierIso)) {
             String[] identifierValue = patientIdentifierIso.split("&");
-            HelperUtil.updateDocumentBy(withDocument,
-                loadTestDataService.getPatientBy(identifierValue[0].replace("^^^", ""), identifierValue[1]));
+            CoreHelpUtils.updateDocumentBy(withDocument,
+                wsService.getPatientBy(identifierValue[0].replace("^^^", ""), identifierValue[1]));
         } else {
-            HelperUtil.updateDocumentBy(withDocument, new Patient());
+            throw new RuntimeException("PatientIdentifier should not be empty.");
```

Need to throw a customized exception